versions.tf

```diff
@@ -1,10 +1,12 @@
 terraform {
+  required_version = ">= 1.0.0"
```

Comment:
I think it's a good idea to add this version constraint, but the CI workflow also needs to be updated to use terraform version 1: https://github.com/equinix/terraform-metal-anthos-on-baremetal/blob/main/.github/workflows/terraform.yml#L22